roles/dns/manage-dns-zones/tasks/named/keys.yml

```diff
@@ -1,8 +1,10 @@
 ---
 
```

Comment:
This seems to all still be happening even when the dnssec variables are set to 'no'.
```
dns_data:
  named_global_config:
    recursion: 'no'
    dnssec_enable: 'no'
    dnssec_validation: 'no'
    dnssec_lookaside: 'no'
```